baker/algolia/algoliaUtils.tsx

```diff
@@ -128,6 +130,7 @@ async function generateWordpressRecords(
                 modifiedDate: post.modifiedDate.toISOString(),
                 content: c,
                 tags: tags.map((t) => t.name),
+                thumbnail: post.thumbnailUrl ?? "/default-thumbnail.jpg",
```

Comment:
I think we'll want to apply `formatUrls()` here, just as we do in `siteRenderers.tsx`.